packages/calypso-e2e/src/lib/blocks/block-flows/slideshow.ts

```diff
@@ -21,7 +21,6 @@ export class SlideshowBlockFlow implements BlockFlow {
 
 	/**
 	 * Constructs an instance of this block flow with data to be used when configuring and validating the block.
-	 *
```

Comment:
the linter didn't like these extra lines for some reason

Reply:
Oh weird, really? My local linter always auto-adds them. 😆 I'm kind of surprised actually!

Reply:
🤔 that's confusing! I ended up reverting this change to use a file that we know got stuck, so I guess that's a problem for another time 😅